src/main/java/com/redhat/devtools/lsp4ij/features/codeLens/LSPCodeLensSupport.java

```diff
@@ -41,14 +40,12 @@
  * </ul>
  */
 public class LSPCodeLensSupport extends AbstractLSPFeatureSupport<CodeLensParams, List<CodeLensData>> {
-    private final CodeLensParams params;

```

Comment:
Parameters must never cached because when you rename file, it uses the same psi file instance, but uri is not the same after the renamed file